balance-management/balance-manager/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+target = "wasm32-wasi"
```

Comment:
please use "wasm32-unknown-unknown" as target instead, as it's the one already used by all the actors